src/pkg/packaging-tools/sharedFramework/sharedFramework.csproj

```diff
@@ -0,0 +1,153 @@
+<Project Sdk="Microsoft.NET.Sdk" InitialTargets="GetPackageReference" DefaultTargets="Restore;Publish">
+  <!--
+    This project lays out a shared framework using the SDK targets.
+  -->
+
+  <UsingTask TaskName="ProcessSharedFrameworkDeps" AssemblyFile="$(LocalBuildToolsTaskDir)core-setup.tasks.dll" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
I don't suppose we have a central place where we define this?

Reply:
There's `NETCoreAppFramework`, will change. Didn't spot this after the move from dn-t.